src/stream/src/executor/aggregation/mod.rs

```diff
@@ -86,6 +93,7 @@ pub trait StreamingAggStateImpl: Any + std::fmt::Debug + DynClone + Send + Sync
         ops: Ops<'_>,
         visibility: Option<&Bitmap>,
         data: &[&ArrayImpl],
+        append_only: bool,
```

Comment:
It looks very verbose to pass `append_only` every time on `apply_batch`... Is it possible to put this attribute in the structs and pass it on `new`?

Reply:
Exactly. We should have different set of aggs for append-only executors. The `try_from` should be refactored to something like `recover_from_state(Datum, AggType, append_only: bool)`.